src/main/java/org/spongepowered/api/plugin/PluginContainer.java

```diff
@@ -59,4 +59,12 @@
      */
     Object getInstance();
 
+    /**
+     * Gets if this PluginContainer wraps something that isn't a 'real' plugin.
```

Comment:
This may refer to the dummy field of the annotation, in which case it should say if it is a dummy plugin.